sound/soc/sof/intel/hda-dai.c

```diff
@@ -751,6 +751,12 @@ static int ipc4_be_dai_trigger(struct snd_pcm_substream *substream,
 	return 0;
 }
 
+static int ipc4_be_dai_trigger(struct snd_pcm_substream *substream,
+			       int cmd, struct snd_soc_dai *dai)
+{
+	return ipc4_sdw_trigger(dai, cmd, substream->stream);
```

Comment:
It will affect DMIC & SSP ?
````
static const struct snd_soc_dai_ops ipc4_dmic_dai_ops = {
	.trigger = ipc4_be_dai_trigger,
};
static const struct snd_soc_dai_ops ipc4_ssp_dai_ops = {
	.trigger = ipc4_be_dai_trigger,
};
````

Reply:
its a bit awkward to call sdw_trigger for other DAI types. Maybe rename ipc4_sdw_trigger() to ipc4_be_common_trigger()?

Reply:
@RanderWang Yes, it is for all DMIC, SSP, and SDW.
@ranj063 Thanks, ipc4_be_common_trigger() is much better than ipc4_sdw_trigger()